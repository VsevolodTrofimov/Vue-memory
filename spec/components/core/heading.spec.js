import { mount, createLocalVue } from '@vue/test-utils'
import { createRenderer } from 'vue-server-renderer'

import coreHeading from '@/src/components/core/Heading/Heading.vue'


const VSSR = createRenderer()
const localVue = createLocalVue()
localVue.config.productionTip = false


describe('Components', () => {
  describe('Core Heading', () => {
    it('matches snapshot', () => {
      const wrapper = mount(coreHeading)

      VSSR.renderToString(wrapper.vm, (err, str) => {
        expect(str).toMatchSnapshot()
      })
    })

    it('changes level', () => {
      const wrapper =  mount(coreHeading, {
        propsData: {level: 1}
      })

      expect(wrapper.contains('h1')).toBe(true)
    })
  })
})
