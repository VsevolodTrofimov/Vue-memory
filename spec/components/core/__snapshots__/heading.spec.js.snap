// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Core Heading matches snapshot 1`] = `<h3 class="heading"> Heading </h3>`;
